csuntechlab/iSTART
# STR-146 -- FE: Implement Information Slides #19, #21, #22, #23, #24 into Alcohol Module

# Description
'Write a meaningful description of your task'
 - I need to implement the informational slides #19, #21, #22, #23 and #24 of the alcohol module.
- Create functionality that will allow us to iterate through these slide numbers
# Testing
'What does the reviewer need to do to test your PR?'
1. Login as `april.feldman`
2. click on any module
3. you will then be redirected to the informational slides
4. there is functionality that doesn't allow the user to click on continue until after 5 seconds of the page
  - make sure that works
  - it would help to look simultaneously look at vuex in devtools
5. you can click back on continue
  - when you click "back" you're able to click on "continue"
6. Review the json file to see that it works and is a good template for the future objects to come
- run `yarn jest-unit`
# Installation
'Are there any dependencies the reviewer needs to download?'
- you might need to run `yarn install` incase you haven't used `jest-extended`
